Revocations, invalid use of undefined type 'struct sshbuf'

Gathering fresh sources for openssh-portable, openssl, and hiba I run into a build error.

Following the instructions on INSTALL.md, I gathered sources 

```
$ git clone git@github.com:openssh/openssh-portable.git
$ git clone git@github.com:sourcequench/hiba.git
$ git clone git://git.openssl.org/openssl.git
```

Building openssl and openssh (with the addition of LD_LIBRARY_PATH=../openssl) worked out fine.

The autogen.sh and configure steps worked fine for hiba, but make ended in sadness - looks like an issue with untyped struct for revocations.

```
make
make  all-am
make[1]: Entering directory '/home/ryanshea/git/hiba'
  CC       libhiba_la-revocations.lo
In file included from revocations.c:15:
revocations.c: In function 'hibagrl_decode':
revocations.c:173:80: error: invalid use of undefined type 'struct sshbuf'
  173 |                 debug3("hibagrl_decode: serial list section content @%zu", blob->off);
      |                                                                                ^~
../openssh-portable/log.h:88:100: note: in definition of macro 'debug3'
   88 | #define debug3(...)             sshlog(__FILE__, __func__, __LINE__, 0, SYSLOG_LEVEL_DEBUG3, NULL, __VA_ARGS__)
      |                                                                                                    ^~~~~~~~~~~
revocations.c:189:75: error: invalid use of undefined type 'struct sshbuf'
  189 |                 debug3("hibagrl_decode: bitmap section content @%zu", blob->off);
      |                                                                           ^~
../openssh-portable/log.h:88:100: note: in definition of macro 'debug3'
   88 | #define debug3(...)             sshlog(__FILE__, __func__, __LINE__, 0, SYSLOG_LEVEL_DEBUG3, NULL, __VA_ARGS__)
      |                                                                                                    ^~~~~~~~~~~
make[1]: *** [Makefile:609: libhiba_la-revocations.lo] Error 1
make[1]: Leaving directory '/home/ryanshea/git/hiba'
make: *** [Makefile:394: all] Error 2
```

--- a/revocations.c
+++ b/revocations.c
@@ -170,7 +170,7 @@ hibagrl_decode(struct hibagrl *grl, struct sshbuf *blob) {
 	debug2("hibagrl_decode: serials_section_size = %" PRIu64, serials_section_size);
 	grl->serials = (const struct serials *)sshbuf_ptr(blob);
 	if (log_level_get() >= SYSLOG_LEVEL_DEBUG3) {
-		debug3("hibagrl_decode: serial list section content @%zu", blob->off);
+		debug3("hibagrl_decode: serial list section content");
 		sshbuf_dump_data(sshbuf_ptr(blob), serials_section_size, stderr);
 	}
 	if ((ret = sshbuf_consume(blob, serials_section_size)) != 0) {
@@ -186,7 +186,7 @@ hibagrl_decode(struct hibagrl *grl, struct sshbuf *blob) {
 	debug2("hibagrl_decode: bitmaps_section_size = %" PRIu64, grl->bitmaps_size);
 	grl->bitmaps = sshbuf_ptr(blob);
 	if (log_level_get() >= SYSLOG_LEVEL_DEBUG3) {
-		debug3("hibagrl_decode: bitmap section content @%zu", blob->off);
+		debug3("hibagrl_decode: bitmap section content");
 		sshbuf_dump_data(sshbuf_ptr(blob), grl->bitmaps_size, stderr);
 	}
 	if ((ret = sshbuf_consume(blob, grl->bitmaps_size)) != 0) {

